### PR TITLE
Initial Studio Docs

### PR DESCRIPTION
**Pull request by rowa97**
meerp

### CODE REVIEW

File: docs/designing-apis/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- How does your API handle errors? 
+
+## How does it fit into Stoplight? 
+
+The Stoplight design module is where you and your team will maintain the single source of truth that describes the inputs and outputs of your API. 
+
+Once you have your API described in the Stoplight design module, you can: 
+
+- Publish all or part of your API 
+- Create API tests from your designs
```
Comment (philsturgeon) [inline docs/designing-apis/getting-started.md]: Can you? Is this talking about the yet-to-be-built Scenarios replacement, or talking about general "you can use these for contract testing if you want" type of stuff.

File: docs/designing-apis/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Publish all or part of your API 
+- Create API tests from your designs 
+- Send requests to your API to debug it 
+- Create a mock API based on your design
+- ...and much more 
+
+## Getting Started 
+
+There are a few ways to get started designing your API with the Stoplight design module:
```
Comment (philsturgeon) [inline docs/designing-apis/getting-started.md]: Why are we saying "Stoplight design module" and not Stoplight Studio?

File: docs/welcome/introduction.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+# Intro
+
+Studio is our next generation app for API design, modeling, and technical writing. 
+
+> A primary goal of Studio is to enrich, not replace, your existing workflows. It works offline, with folders and files on your computer, just like your favorite IDEs.
+
+Here is some of what it can do:
+
+- **OpenAPI v2 AND v3:** Form based design mode means you don't need to be an OpenAPI expert to get started. Studio also supports OpenAPI autocomplete in write mode, a read mode to visualize HTTP operations and models, mocking, and linting for OpenAPI v2 and v3.
+- **Standalone JSON Schema Modeling:** Studio encourages you to split your models into separate files, and then makes it easy to create `$refs` between them.
```
Comment (philsturgeon) [inline docs/welcome/introduction.md]: This seems like its confusing two different things.

1. We encourage and support OpenAPI users splitting their models out into files with $refs
2. We support standalone JSON Schema completely outside of an OpenAPI context for people who do that

1 is not JSON Schema its "OpenAPI Schema Objects" and confusing those things is asking for trouble.

File: docs/welcome/introduction.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+- **Stoplight Flavored Markdown:** SMD is an optional, lightweight extension to regular markdown. It enables a few advanced features such as tabs and callouts.
```
Comment (philsturgeon) [inline docs/welcome/introduction.md]: This sounds scary and would worry me if i was about to use it. Its not really a special flavour we just have some conventions around code block names and certain meanings, right? It's still the same syntax, just with extra meaning?

Also can we mention if we are Commonmark based, or which flavour we are based on? 

Markdown is a mess: https://phil.tech/markdown/2014/11/30/state-of-markdown/

File: docs/designing-apis/getting-started.md (hunk 1 above)
Comment (rowa97) [reply docs/designing-apis/getting-started.md]: this section was pulled from our previous content as filler, what features do you think we should highlight here? what will you be able to do with explorer? is it worth mentioning in Studio docs?

File: docs/designing-apis/getting-started.md (hunk 2 above)
Comment (rowa97) [reply docs/designing-apis/getting-started.md]: updated

File: docs/welcome/introduction.md (hunk 3 above)
Comment (rowa97) [reply docs/welcome/introduction.md]: @marbemac review

File: docs/welcome/introduction.md (hunk 4 above)
Comment (rowa97) [reply docs/welcome/introduction.md]: @marbemac review

File: docs/designing-apis/getting-started.md (hunk 1 above)
Comment (philsturgeon) [reply docs/designing-apis/getting-started.md]: Ah gotcha, well we removed Scenarios so lets hide this. We can add a thing about spectral after mocking on line 26:

- Get feedback in the form of validation and style warnings

**Comment by philsturgeon** [comment]
Is this one going anywhere or is #32 replacing it? or both?

**Comment by rossmcdonald** [comment]
@philsturgeon #32 is what we will be merging next week for the release. This branch has a few extra articles that I'd like to take more time to sift through and review before adding them to the official docs. Any updates should be branched from #32 though.

BTW I did pull your mocking article into #32, so that one is included. #32 also has an updated file layout based on the latest changes sidebar navigation changes (it has a root README, for example).

**Comment by philsturgeon** [comment]
Should this be closed down now? Seems like it might have been superseded by #32 @rowa97 @rossmcdonald

**Comment by rossmcdonald** [comment]
Closing since this is no longer in a merge-able state.

@rowa97 Can you create another PR with the articles that didn't make it into #32 (masking, for example)?

**Pull request closed without merging**